docs/ref/contrib/contenttypes.txt

```diff
@@ -299,6 +299,33 @@ model:
        is ``True``. This mirrors the ``for_concrete_model`` argument to
        :meth:`~django.contrib.contenttypes.models.ContentTypeManager.get_for_model`.
 
+    .. attribute:: GenericForeignKey.allow_unsaved_instance_assignment
+
+        Django prevents unsaved model instances from being assigned to a
```

Comment:
I'd rather simply link to `ForeignKey.allow_unsaved_instance_assignment` as `OneToOne` field does. I don't expect it to be commonly used so I think it'd be better not to make a big deal about it.